src/utils/tana-formatter/field-formatting.ts

```
@@ -0,0 +1,216 @@
+
+/**
+ * Format simple lines as parent/child structure
+ */
+export function formatLinesAsHierarchy(lines: string[]): string[] {
+  if (!lines || lines.length === 0) {
+    return []
+  }
+
+  const filteredLines = lines.filter((line) => {
+    // Remove all Unicode whitespace and invisible characters
+    const cleaned = line.replace(/[\s\u200B\u200C\u200D\uFEFF]/g, '')
```

Biome (1.9.4)

[error] A character class cannot match a joined character sequence.

A zero width joiner composes several emojis into a new one. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)